{{cookiecutter.project_name}}/tasks.py

```diff
@@ -317,25 +326,46 @@ def update(c: Context):
     install(c, pip_args="--upgrade", msg=False)
 
 
-@task
-def test(c: Context):
+@task(iterable="pytest_args")
```

Comment:
Ahh i see - I would probably have it as just a string. No reason to do the parsing.